### PR TITLE
Duplicate removal about invoice print and adding the "Payment actions" block

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

1 Case:  Add the "Payment actions" block 

2 Case: Since "Printing Invoices" is described in a separate subcategory, it should not be described in the "Creating an Invoice" topic, this only loads the document. This PR removes unnecessary duplicate information
image

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/invoice-create.html
- https://docs.magento.com/user-guide/sales/refunds-customer-account.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=refund

### CODE REVIEW

**Comment by dobooth** [comment]
Any chance you could remove that Acrobat screenshot from what appears to be 1998 from the Printing Invoices topic?

**Comment by EvheniiaKolkhiieva** [comment]
@dobooth yes, for sure. 

Should I just delete the image in the additional commit or add an updated modern one?

**Comment by EvheniiaKolkhiieva** [comment]
Hi @dobooth,
I have deleted the Acrobat screenshot from the Printing invoices topic.
Could you please approve?

Thank you!

File: src/sales/refunds-customer-account.md [hunk 1]
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
 
    ![]({% link images/images-ee/customer-account-store-credit.png %}){: .zoom}
    _Amount refunded to store credit_
+
+## Payment actions
+
+You can configure payment actions for your specific [payment method]({% link configuration/sales/payment-methods.md %}). Each payment method has a different set of payment actions.
+
+|Payment Action |Description
```
Comment (leslietilling) [inline src/sales/refunds-customer-account.md:37]: Sentence case for column headings

```suggestion
|Payment action |Description
```

File: src/sales/refunds-customer-account.md [hunk 2]
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
...
+|--- |---
+|Capture Online |When the invoice is submitted, the system captures the payment from the third-party payment gateway. You have the ability to create a credit memo and void the invoice.
+|Capture Offline |When the invoice is submitted, the system does not capture the payment. It is assumed that the payment is captured directly through the gateway, and you no longer have the option to capture this payment through Magento. You have the ability to create a credit memo, but you do not have the option to void the invoice. (Even though the order used an online payment, the invoice is essentially an offline invoice.)
+|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a Capture button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing you are able to create a credit memo and void the invoice.
```
Comment (leslietilling) [inline src/sales/refunds-customer-account.md:41]:
```suggestion
|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a _Capture_ button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing, you are able to create a credit memo and void the invoice.
```

File: src/sales/refunds-customer-account.md [hunk 3]
```diff
@@ -29,3 +29,16 @@ If a refund has been issued for an order, customers can view the refund informat
...
+
+{:.bs-callout-info}
+Do not select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the Capture button.
```
Comment (leslietilling) [inline src/sales/refunds-customer-account.md:44]:
```suggestion
**_Do not_** select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the _Capture_ button.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a few minor edits and this one looks good.

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling,

I have implemented your edits, could you please approve? 

Thank you!

Re: pay attention, seems the "Adding the "Payment actions block " commit mistakenly got into this PR, since it does not apply to the description above, but I updated the description to combine both edits

File: src/sales/refunds-customer-account.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ☑️

File: src/sales/refunds-customer-account.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ☑️

File: src/sales/refunds-customer-account.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/refunds-customer-account.md]: Done ☑️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
I will fix the conflict in the team fork.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1189`.

**Comment by EvheniiaKolkhiieva** [comment]
Thank you a lot @leslietilling ☺️

**Pull request merged**